src/workerd/api/node/crypto.h

```diff
@@ -7,6 +7,48 @@ namespace workerd::api::node {
 
 class CryptoImpl final: public jsg::Object {
 public:
+  // DH
+  class DHHandle final: public jsg::Object {
```

Comment:
`DhHandle` would be more idiomatic for our codebase. I prefer the non-abbreviated name tho: `DiffieHellmanHandle`